components/Header.ts

```diff
@@ -0,0 +1,6 @@
+
+function Header(x: number, y: number): number {
+    return x + y;
+}
+
+console.log(Header(2,2));
```

Comment:
Codacy has a fix for the issue: [Insert `·`](https://app.codacy.com/gh/thamizhMsys/testRepo6/pullRequest?prid=9650719)
```suggestion
console.log(Header(2, 2));
```